tests/main_pipeline_alevin.nf.test

```diff
@@ -28,33 +28,34 @@ nextflow_pipeline {
                 {assert workflow.trace.tasks().size() == 15},
 
                 // How many results were produced?
-                {assert path("${outputDir}/results_alevin").list().size() == 5},
-                {assert path("${outputDir}/results_alevin/alevin").list().size() == 3},
-                {assert path("${outputDir}/results_alevin/alevin/mtx_conversions").list().size() == 5},
-                {assert path("${outputDir}/results_alevin/alevinqc").list().size() == 2},
-                {assert path("${outputDir}/results_alevin/fastqc").list().size() == 12},
-                {assert path("${outputDir}/results_alevin/multiqc").list().size() == 3},
+                {assert path("${outputDir}/results_simpleaf").list().size() == 4},
+                {assert path("${outputDir}/results_simpleaf/simpleaf").list().size() == 3},
+                {assert path("${outputDir}/results_simpleaf/simpleaf/mtx_conversions").list().size() == 5},
+                {assert path("${outputDir}/results_simpleaf/fastqc").list().size() == 12},
+                {assert path("${outputDir}/results_simpleaf/multiqc").list().size() == 3},
 
                 //
                 // Check if files were produced
                 //
-                {assert new File( "${outputDir}/results_alevin/alevin/mtx_conversions/Sample_X/Sample_X_raw_matrix.h5ad" ).exists()},
-                {assert new File( "${outputDir}/results_alevin/alevin/mtx_conversions/Sample_Y/Sample_Y_raw_matrix.h5ad" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/mtx_conversions/Sample_X/Sample_X_raw_matrix.h5ad" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/Sample_X/simpleaf_quant/af_quant/alevin/quants_mat_cols.txt" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/Sample_X/simpleaf_quant/af_quant/alevin/quants_mat_rows.txt" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/Sample_X/simpleaf_quant/af_quant/alevin/quants_mat.mtx" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/Sample_X/simpleaf_quant/af_quant/alevin/quants.h5ad" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/mtx_conversions/Sample_Y/Sample_Y_raw_matrix.h5ad" ).exists()},
+                {assert new File( "${outputDir}/results_simpleaf/simpleaf/Sample_Y/simpleaf_quant/af_quant/alevin/quants_mat_cols.txt" ).exists()},
```

Comment:
is any of these alevin files now possible to have as snaps with the new version?
I remember before they could vary the sorting but not sure about the newest version.

Reply:
Hmm. Generally, I do not think is necessary. But, if it would be possible to have it being part of the snaps, it would surely add robustness.

@grst , do you think this should be here or is the PR already big enough and better to have another?

Reply:
I mean having a snapshot is obviously better than not having one, but I won't insist on it.